news/2017-09-03-qat-gazette-september-2017.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "The QAT Gazette: September 2017"
+date: 2017-09-03 09:00:00 +0000
+---
+
+# QAT Gazette - August 2017
```

Comment:
drop this title, post headers does it for you

Reply:
I see, done